demo/image/placeholder/playground

```diff
@@ -0,0 +1,34 @@
+const placeholder= (
+  <svg viewBox="0 0 1 1">
+      <rect width="100%" height="100%" />
+  </svg>
+)
+
+return (
+  <div>
+    <h2>Image Placeholder</h2>
+    <div style={{width: '20%'}}>
+      <ImagePlaceholder
```

Comment:
Better style in CSS class

Reply:
It's just the demo so its easier to see what happens